src/lib/updating-element.ts

```diff
@@ -151,8 +151,9 @@ export const defaultConverter: ComplexAttributeConverter = {
       case Number:
         return value === null ? null : Number(value);
       case Object:
+        return value === '' ? {} : JSON.parse(value!);
```

Comment:
I think rather than this, which wouldn't fix the demo linked in the issue, we want to put a try/catch around the `JSON.parse()`.